src/legacy/core_plugins/kibana/public/discover/np_ready/components/field_chooser/discover_index_pattern.test.tsx

```diff
@@ -42,7 +48,7 @@ const indexPattern2 = {
 
 const defaultProps = {
   indexPatternList: [indexPattern1, indexPattern2],
-  selectedIndexPattern: indexPattern1,
+  selectedIndexPattern: indexPattern,
```

Comment:
Are you intentionally using an index pattern that's not in the indexPatternList?

Reply:
`selectedIndexPattern` is of type `IIndexPattern`, `indexPatternList` of type `SavedObject` 
I've removed some code that extracted the  `selectedIndexPattern` of the list of `SavedObject`. that's why I had to adapt here.

Reply:
Note: it would be nice to have a list entries of type IIndexPattern, but AFAIK to get this, it would need some additional changes